### ISSUE TITLE
Get parameter array handled incorrectly

### ISSUE THREAD

**Issue by k-webdesign** [MetaModels/core#1498]
## My environment is:

(Please fill in the actual values from your environment - check Contao Manager or use `composer show`)

| Key                                 | Value    | Comments                         |
| ----------------------------------- | ---------| ---------------------------------|
| PHP version:                        |   7.4       |                                  |
| Contao version:                     |   4.9       |                                  |
| MetaModels version:                 |    2.2      |                                  |

## Issue description

```
TypeError:
Argument 2 passed to MetaModels\Filter\FilterUrl::setGet() must be of the type string, array given, called in /.../vendor/metamodels/core/src/Filter/FilterUrlBuilder.php on line 204

  at vendor/metamodels/core/src/Filter/FilterUrl.php:147
  at MetaModels\Filter\FilterUrl->setGet()
     (vendor/metamodels/core/src/Filter/FilterUrlBuilder.php:204)
  at MetaModels\Filter\FilterUrlBuilder->addFromCurrentRequest()
     (vendor/metamodels/core/src/Helper/PaginationLimitCalculator.php:185)
  at MetaModels\Helper\PaginationLimitCalculator->__construct()
     (vendor/metamodels/core/src/ItemList.php:199)
  at MetaModels\ItemList->__construct()
     (vendor/metamodels/core/src/CoreBundle/Controller/ListControllerTrait.php:141)
  at MetaModels\CoreBundle\Controller\FrontendModule\ItemListController->getResponseInternal()
     (vendor/metamodels/core/src/CoreBundle/Controller/FrontendModule/ItemListController.php:96)
  at MetaModels\CoreBundle\Controller\FrontendModule\ItemListController->getResponse()
     (vendor/contao/core-bundle/src/Controller/FrontendModule/AbstractFrontendModuleController.php:40)
  at Contao\CoreBundle\Controller\FrontendModule\AbstractFrontendModuleController->__invoke()
     (vendor/metamodels/core/src/CoreBundle/Controller/FrontendModule/ItemListController.php:64)
  at MetaModels\CoreBundle\Controller\FrontendModule\ItemListController->__invoke()
     (vendor/symfony/http-kernel/HttpKernel.php:169)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw()
     (vendor/symfony/http-kernel/HttpKernel.php:81)
  at Symfony\Component\HttpKernel\HttpKernel->handle()
     (vendor/symfony/http-kernel/HttpCache/SubRequestHandler.php:85)
  at Symfony\Component\HttpKernel\HttpCache\SubRequestHandler::handle()
     (vendor/symfony/http-kernel/Fragment/InlineFragmentRenderer.php:81)
  at Symfony\Component\HttpKernel\Fragment\InlineFragmentRenderer->render()
     (vendor/symfony/http-kernel/Fragment/FragmentHandler.php:85)
  at Symfony\Component\HttpKernel\Fragment\FragmentHandler->render()
     (vendor/contao/core-bundle/src/Fragment/FragmentHandler.php:88)
  at Contao\CoreBundle\Fragment\FragmentHandler->render()
     (vendor/contao/core-bundle/src/Resources/contao/modules/ModuleProxy.php:62)
  at Contao\ModuleProxy->generate()
     (vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:443)
  at Contao\Controller::getFrontendModule()
     (vendor/contao/core-bundle/src/Resources/contao/pages/PageRegular.php:173)
  at Contao\PageRegular->prepare()
     (vendor/contao/core-bundle/src/Resources/contao/pages/PageRegular.php:53)
  at Contao\PageRegular->getResponse()
     (vendor/contao/core-bundle/src/Resources/contao/controllers/FrontendIndex.php:343)
  at Contao\FrontendIndex->renderPage()
     (vendor/symfony/http-kernel/HttpKernel.php:169)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw()
     (vendor/symfony/http-kernel/HttpKernel.php:81)
  at Symfony\Component\HttpKernel\HttpKernel->handle()
     (vendor/symfony/http-kernel/Kernel.php:201)
  at Symfony\Component\HttpKernel\Kernel->handle()
     (web/preview.php:33)                
```

## Steps to reproduce

suffix url with get parameter (e.g. `?types[]=PART_TIME`)

**Comment by zonky2** [comment]
@k-webdesign pls test the fix https://github.com/MetaModels/core/pull/1499

**Comment by zonky2** [comment]
fixed in 2.2.9
